debian/portal.triggers

```diff
@@ -1,7 +1,7 @@
 # Register interest in Python interpreter changes (Python 2 for now); and
```

Comment:
dated comment (Python 2), but not urgent.

Reply:
Thanks! fixed in 8be5066